### PR TITLE
feat(wallet): change format of values to show only 8 digits 

### PR DESCRIPTION
**Pull request by AndresQuijano**
image

image

image

[Jira](https://rsklabs.atlassian.net/browse/ROLLUP-410)

### CODE REVIEW

File: src/components/ChooseToken.vue [hunk 1]
```diff
@@ -202,6 +204,9 @@ export default Vue.extend({
     displayTokenBalance(): boolean {
       return !this.onlyMintTokens && (this.tokensType === "L1-Tokens" || this.tokensType === "L2-Tokens");
     },
+    provider(): RestProvider {
```
Comment (sergio-jara) [inline src/components/ChooseToken.vue:207]: what's the purpose of this?

Comment (AndresQuijano) [reply src/components/ChooseToken.vue:207]: I need the provider to perform [this](https://github.com/rsksmart/rif-rollup-wallet/pull/143/files#diff-be1ce0dcd6f3e04c3beb212a7c84e826c72060287811b74232a61349ff530628R229) call.

File: src/components/ChooseToken.vue [hunk 2]
```diff
@@ -220,8 +225,10 @@ export default Vue.extend({
       }
       return this.$emit("chosen", symbolOrID);
     },
-    parseBigNumber(value: string) {
-      return formatFixed(value, 18);
```
Comment (LordCheta) [inline src/components/ChooseToken.vue:224]: Isn't this approach better as it doesn't rely on importing new dependencies into the project?

**Review by sergio-jara — APPROVED** [review]

File: src/components/ChooseToken.vue (hunk 2 above)
Comment (LordCheta) [reply src/components/ChooseToken.vue:224]: Being that, it's safer to use a proper Big number library for this kind of operations, I believe we can move forward

**Pull request merged**